python/tchannel/rw.py

```diff
@@ -347,7 +347,7 @@ class NumberReadWriter(ReadWriter):
         1: '>B',
         2: '>H',
         4: '>I',
-        8: '>Q',
+        8: '>q',
```

Comment:
I don't know if you want to change all parsed 64-bit integers to signed. Can we make a separate `rw.number` parameter `signed=True/False`?